client/src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -83,13 +83,7 @@ const FeedbackForm = () => {
                 color="primary"
                 disabled={!dirty || !isValid || isSubmitting || creating}
               >
-                {creating && (
-                  <Spinner
-                    size="sm"
-                    color="white"
-                    data-testid="creating-spinner"
```

Comment:
I think a unit testing is failing because this was removed, see [here](https://github.com/TACC/Frontera-Portal/pull/331/checks?check_run_id=1780857944#step:6:845)

Alternatively, we can query for the hidden `Loading...` text instead (used [here](https://github.com/TACC/Frontera-Portal/pull/331/checks?check_run_id=1780857944#step:6:965)) using [`getByText`](https://testing-library.com/docs/queries/about/#textmatch)

Reply:
Oh that's odd, I think I removed that by accident when trying to remove the className I had put there previously...